src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,11 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD, ElementType.TYPE})
```

Comment:
The `@Inject` annotation should not target `ElementType.TYPE`. It should only target `ElementType.FIELD` or `ElementType.METHOD` for proper dependency injection. Please remove `ElementType.TYPE` from the target.